.mailmap

```diff
@@ -0,0 +1,5 @@
+Andrew Kofink <ajkofink@gmail.com> <akofink@redhat.com>
```

Comment:
@akofink please review which address you want as primary

Reply:
ajkofink@gmail.com please :)

Reply:
great, thanks!